scripts/metadata_operations.py

```diff
@@ -187,6 +208,34 @@ def anonymize_sample_ids(input_output_md, input_output_tab):
     md.to_csv(input_output_md, sep='\t', index=False, header=True)
 
 
+@cli.command()
+@click.option('--input-output-md', type=click.Path(exists=True), required=True)
+@click.option('--input-output-tab', type=click.Path(exists=True),
+              required=True)
+def normalize_gender_and_sex(input_output_md, input_output_tab):
+    """Normalize gender_v2 and sex columns
+
+    10317 has historically used the 'sex' field to reflect a participant's
+    gender. Post-relaunch, we reworded the question and are using 'gender_v2'
+    to store the response to the new question. While we want to keep these
+    separate for analysis purposes, we're going to temporarily normalize them
+    to generate reports for users. The least invasive way to do this is to
+    copy gender_v2 responses into sex for any participants who have a valid
+    response to gender_v2.
+    """
+    md = pd.read_csv(input_output_md, sep='\t', dtype=str)
+    tab = biom.load_table(input_output_tab)
+
+    mapping = dict()
+    mapping.update(_normalize_gender_and_sex(md))
+
+    tab.update_ids(mapping, inplace=True, strict=False)
```

Comment:
Sample IDs are changing in the feature table?

Reply:
Removed